Include/internal/pycore_bitutils.h

```diff
@@ -82,6 +84,45 @@ _Py_bswap64(uint64_t word)
 }
 
 
+// Population count: count the number of 1's in 'u'
```

Comment:
Replace `'u'` with `'x'` here?

Reply:
Oops, right. I forgot to update the comment when I renamed the parameter.